bin/changelog.ts

```diff
@@ -31,16 +28,13 @@ void (async () => {
   const from = until ? newProductionTag : lastProductionTag;
   const to = until ? until : newProductionTag;
 
-  console.info(`Generating changelog with commits from "${from}" to "${to}".`);
-
   try {
     const changelog = await Changelog.generate({
       exclude: ['chore', 'docs', 'refactor', 'style', 'test', 'runfix'],
       repoUrl: pkg.repository.url.replace('.git', ''),
       tag: `${from}...${to}`,
     });
-    fs.outputFileSync(outputPath, changelog, 'utf8');
```

Comment:
dont we need to output the changelog?

Reply:
it's now ouputed to the console rather than a file (and the caller can decide on which file they want to output it)

Reply:
see https://github.com/wireapp/wire-webapp/pull/16379/files/65b29b80b760f7b9a451e0fdb395b6205cbe0c26#diff-551d1fcf87f78cc3bc18a7b332a4dc5d8773a512062df881c5aba28a6f5c48d7R122 for an exemple of how to use it.

Baseline is:

`yarn changelog:production > file.md`